api/restHandler/UserRestHandler.go

```diff
@@ -167,6 +194,32 @@ func (handler UserRestHandlerImpl) UpdateUser(w http.ResponseWriter, r *http.Req
 			return
 		}
 	}
+
+	// auth check inside groups
+	if len(userInfo.Groups) > 0 {
```

Comment:
this code and code from line 114 to 137 is exact duplicate... please extract it into a method

Reply:
No duplicate, code fixed, rbac.ActionCreate to rbac.ActionUpdate